documentation/release-notes-2-9.md

```diff
@@ -0,0 +1,2230 @@
+to buy a support contract for one or more systems.
+Buying a support contract actually buys time for the core developers to work on ReaR.
+
+See the Relax-and-Recover website support [relax-and-recover.org/support](/support)
+
+## Supported and Unsupported Operating Systems
+
+We try to keep our "Test Matrix" wiki pages up-to-date with feedback we receive from the community.
+For example for ReaR 2.9 see <https://github.com/rear/rear/wiki/Test-Matrix-ReaR-2.9>
+
+ReaR 2.9 is supported on the following Linux operating systems:
+
+* Fedora 40, 41 (Silverblue not supported)
+* RHEL 7, 8, and 9
+* RHEL clones (CentOS, Scientific Linux, Alma Linux, Oracle Linux) 7, 8, and 9
+* SLES 15
+* openSUSE Leap 15.x
+* Debian 10 and 11
+* Ubuntu 20.04, 22.04, 24.04
+* Arch Linux / Manjaro
+
+ReaR 2.8 dropped official support for the following Linux operating systems:
+
+* Fedora < 39
+* RHEL < 7
+* RHEL clones (CentOS, Scientific Linux, Alma Linux, Oracle Linux) < 7
+* SLES < 15
+* openSUSE Leap < 15.x and older openSUSE versions
+* openSUSE Tumbleweed
+* Debian < 10
+* Ubuntu < 20
+
+Usually ReaR 2.9 should also work on newer versions of the above listed supported Linux operating systems
+but sometimes arbitrary failures can happen when software that is used by ReaR
+(like partitioning tools, filesystem tools, bootloader tools, ISO image creating tools, networking tools, and so on)
+changes in not fully backward compatible ways or when there are innovations of the basic system
+(like kernel, storage, bootloader, init, networking, and so on)
+that are not yet supported by ReaR.
+
+In theory ReaR 2.9 should work on openSUSE Tumbleweed
+but in practice arbitrary failures could happen at any time
+because the Tumbleweed distribution is a pure rolling release version of openSUSE
+containing the latest stable versions of all software
+(cf. <https://en.opensuse.org/Portal:Tumbleweed>)
+so arbitrary changes of any software are possible at any time
+that could arbitrarily break how ReaR works.
+
+ReaR 2.9 may still work for SLES < 15 and openSUSE Leap < 15.x
+but it is no longer tested there so arbitrary regressions could appear,
+in particular on systems with Bash before version 4.
+
+ReaR 2.9 and earlier versions are known to no longer work
+for the following Linux operating systems:
+
+* RHEL 5 (and probably also CentOS 5): See issue #1766
+* SLES 9 and 10: See issue #1842
+
+If you need support for an unsupported Linux operating system you must acquire a ReaR support contract.
+
+Requests to port ReaR to another operating system (not Linux) can only be achieved with serious sponsoring.
+
+## Supported and Unsupported Architectures
+
+ReaR 2.9 is supported on:
+
+* Intel x86 (32bit and 64bit) architectures
+* AMD x86 (64bit) architecture
+* PPC64LE architecture
+
+ReaR 2.9 may or may not work on:
+
+* PPC64 processors
+* Intel Itanium processors
+* ARM type of processors
+* IBM Z "s390x" type of processors
+
+ReaR 2.9 is known to not support:
```

Comment:
Perhaps rephrase this to "ReaR 2.9 does not support:"

Reply:
I copied it verbatim from the 2.8 release notes, where it originated in https://github.com/rear/rear.github.com/commit/24251c3f092fd2ceca9082def5a052a72326eaad . I suppose the reason for the wording could be that "ReaR 2.9 does not support:" might sound like a definitive list, while "ReaR 2.9 is known to not support" makes clear that there are others, we just have not cared that they are not supported.

Reply:
I agree with @pcahyna so I think this conversation can be solved
with "keep it as is".